tests/test_model_frame.py

```diff
@@ -0,0 +1,11 @@
+from compas_fab.robots import RobotCellLibrary
+
+rc, rcs = RobotCellLibrary.ur10e()
+model = rc.robot_model
+for link in model.iter_links():
+    print(link.name)
+    print(model.get_link_visual_meshes(link))
+    print(model.get_link_collision_meshes(link))
+    print(model.get_link_visual_meshes_joined(link))
+    print(model.get_link_collision_meshes_joined(link))
+    print("---------------------")
```

Comment:
love it that there's a test, but maybe replace the print (which iirc do not appear in the pytest output) with some meaningful assertions?

Reply:
ah also while this is in the `tests` directory it does not adhere to the structure of a unittest AKAK functions starting with `test_...`

Reply:
Oh, no this is a file I did not intend to upload.
The robot with geometry is very slow to run on CI, there is nothing like this currently in the other tests.
I think I have removed this file in the branch head.

Reply:
Oh no, you are not supposed to see this. Please un-see the idea of the test.

Reply:
<test unseeing accomplished>